math/average.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <assert.h>
+
+//average of an array
+
+double average(int avg[]) {
+    double result = 0;
+    int size=0;
+    
+    while(avg[size+1]!='\0'){
+        size++;
+    }
+
+    for (int i = 0; i < size; i++) {
+        result += avg[i];
+    }
+
+    return result / size;
```

Comment:
what if the size is zero?

Reply:
lets do something like if (size== 0) {
        perror("empty array");
    }